storage/innobase/handler/i_s.cc

```diff
@@ -2744,7 +2744,8 @@ i_s_fts_index_cache_fill(
 		DBUG_RETURN(0);
 	}
 
-	if (!user_table->fts || !user_table->fts->cache) {
+	if (!user_table->fts || !user_table->fts->cache
+	    || !user_table->is_readable()) {
 		dict_table_close(user_table, FALSE, FALSE);
 		goto no_fts;
 	}
```

Comment:
Would `is_accessible()` be a better choice than `is_readable()`? Which checks are being performed in `dict_table_open_on_id()` already?